catalog/nsili/catalog-nsili-common/src/main/java/org/codice/alliance/nsili/common/NsiliExtendSecurityAttributes.java

```diff
@@ -0,0 +1,86 @@
+/**
+ *  Copyright (c) Codice Foundation
+ *  <p>
+ *  This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ *  General Public License as published by the Free Software Foundation, either version 3 of the
+ *  License, or any later version.
+ *  <p>
+ *  This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ *  even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ *  Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ *  is distributed along with this program and can be found at
+ *  <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ */
+package org.codice.alliance.nsili.common;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import ddf.catalog.data.AttributeDescriptor;
+import ddf.catalog.data.MetacardType;
+import ddf.catalog.data.impl.AttributeDescriptorImpl;
+import ddf.catalog.data.impl.BasicTypes;
+
+public class NsiliExtendSecurityAttributes implements NsiliExtendedSecurity, MetacardType {
+    private static final Set<AttributeDescriptor> DESCRIPTORS = new HashSet<>();
+
+    private static final String NAME = "nsili.security";
+
+    static {
+        DESCRIPTORS.add(new AttributeDescriptorImpl(EXT_SECURITY_METADATA_CLASSIFICATION,
```

Comment:
The MGMP PR (https://github.com/codice/alliance/pull/110) adds these to the SecurityAttributes in Catalog Core, if that's merged beforehand, this can be removed.